build/build.sh

```diff
@@ -111,6 +117,17 @@ function InstallDotNetCli {
   DotNetCliVersion="$( GetVersionsPropsVersion DotNetCliVersion )"
   DotNetInstallVerbosity=""
 
+  if $dogfood
+  then
+    export SDK_REPO_ROOT="$RepoRoot"
+    export SDK_CLI_VERSION="$DotNetCliVersion"
+    export MSBuildSDKsPath="$ArtifactsConfigurationDir/bin/Sdks"
+    export DOTNET_MSBUILD_SDK_RESOLVER_SDKS_DIR="$MSBuildSDKsPath"
+    export NETCoreSdkBundledVersionsProps="$DotNetRoot/sdk/$DotNetCliVersion/Microsoft.NETCoreSdk.BundledVersions.props"
+    export CustomAfterMicrosoftCommonTargets="$MSBuildSDKsPath/Microsoft.NET.Build.Extensions/msbuildExtensions-ver/Microsoft.Common.Targets/ImportAfter/Microsoft.NET.Build.Extensions.targets"
+    export MicrosoftNETBuildExtensionsTargets="$CustomAfterMicrosoftCommonTargets"
```

Comment:
I want dotnet that was downloaded on my PATH too in this env.

Reply:
Its added to the path already down below